astropy/config/tests/data/suppress.cfg

```diff
@@ -0,0 +1,2 @@
+[utils.data]
+dataurl = http://example.org/astropy/suppress_config/
```

Comment:
Is this a misuse of `dataurl`? It appears to be pointing to remote data server, not to config files. What if I still want to use data from that server?